packages/zendesk-adapter/src/filters/article/utils.ts

```
@@ -226,17 +226,21 @@ export const updateArticleTranslationBody = async ({
     .filter(isReferenceExpression)
     .filter(translationInstance => isTemplateExpression(translationInstance.value.value.body))
     .forEach(async translationInstance => {
-      replaceTemplatesWithValues(
-        { values: [translationInstance.value.value], fieldName: 'body' },
-        {},
-        (part: ReferenceExpression) => {
-          const attachmentIndex = attachmentElementsNames.findIndex(name => name === part.elemID.name)
-          if (attachmentIndex !== -1) {
-            return attachmentInstances[attachmentIndex].value.id.toString()
+      try {
+        replaceTemplatesWithValues(
+          { values: [translationInstance.value.value], fieldName: 'body' },
+          {},
+          (part: ReferenceExpression) => {
+            const attachmentIndex = attachmentElementsNames.findIndex(name => name === part.elemID.name)
+            if (attachmentIndex !== -1) {
+              return attachmentInstances[attachmentIndex].value.id.toString()
+            }
+            return prepRef(part)
           }
-          return prepRef(part)
-        }
-      )
+        )
+      } catch (e) {
+        log.error(`Error serializing article translation body in Deployment for ${translationInstance.elemID.getFullName()}: ${e}, stack: ${e.stack}`)
```

shouldn't we return an error here? I'm guessing the `put` request will fail, no?

---

I was thinking we would catch these in `adapter.ts` and convert to errors (`preDeploy` is an implementation detail but not exposed externally), I think we are just missing this code